CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
-# Upcoming
+## 0.23.0 / 2018-08-06
 
+* [FEATURE] Introduce new flag to control logging format (#1475)
+* [FEATURE] Ensure Prometheus Operator container runs as `nobody` user by default (#1393)
+* [BUGFIX] Fix reconciliation of Prometheus StatefulSets due to ServiceMonitors and PrometheusRules changes when a single namespace is being watched (#1749)
 * [CHANGE] Deprecate specification of Prometheus rules via ConfigMaps in favor of `PrometheusRule` CRDs
```

Comment:
Can you order the entries like:

```
Entries in the CHANGELOG.md are meant to be in this order:

    [CHANGE]
    [FEATURE]
    [ENHANCEMENT]
    [BUGFIX]
```

https://github.com/prometheus/prometheus/wiki/HOWTO-cut-a-new-release